docs/evaluation.md

```diff
@@ -55,7 +59,7 @@ Then navigate to `localhost:4000/evaluate`.
 Alternatively, you can provide an output file to inspect the output in a json file.
 
 ```posix-terminal
-genkit eval:flow bobQA --input testQuestions.json --output eval-result.json
```

Comment:
There is another reference to `bobQA` much further down below, under "Running on existing datasets"